kannada/kannada_kali/kannada_kali.md

```diff
@@ -1 +1,3 @@
-ಕನ್ನಡ ಕಲಿಯಲು [ಇದು](https://play.google.com/store/apps/details?id=com.pgjpb.learnkannada&hl=en)  ಒಳ್ಳೆಯ ಸಾಧನ.
\ No newline at end of file
+ಕನ್ನಡ ಕಲಿಯಲು [ಇದು](https://play.google.com/store/apps/details?id=com.pgjpb.learnkannada&hl=en)  ಒಳ್ಳೆಯ ಸಾಧನ.
+
+ಅದರ ಜೊತೆಗೆ [ಈ]()https://play.google.com/store/apps/details?id=com.sriandroid.justkannada)ಸಾಧನದಿಂದ ಕನ್ನಡದಲ್ಲಿ ಟೈಪ್ ಮಾಡಬಹುದು
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.